ci/goreleaser/goreleaser.yml

```diff
@@ -79,56 +70,43 @@ dockers:
     - "coprocess"
     - "tyk.conf.example"
 
-# Build tykio/xDH_REPO, cloudsmith/xCOMPATIBILITY_NAME (arm64)
+# Build gateway hybrid container amd64
 - ids:
     - std
   image_templates:
-    - "tykio/tyk-gateway:{{ .Tag }}-arm64"
-    - "docker.tyk.io/tyk-gateway/tyk-gateway:{{ .Tag }}-arm64"
```

Comment:
If this is removed, how will cloudsmith get updated?

Reply:
I think it's because the dockers order was changed, and the diff is not clear, the cloudsmith image template reference for arm64 is here.

https://github.com/TykTechnologies/tyk/blob/dca1e449e62b5faf0a1ddb32873b207dff0b246b/ci/goreleaser/goreleaser.yml#L95